modules/efk-logging-elasticsearch-tolerations.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * logging/efk-logging-elasticsearch.adoc
+
+[id="efk-logging-elasticsearch-tolerations_{context}"]
+= Using tolerations to control the Elasticsearch pod placement
+
+You can ensure which nodes the Elasticsearch pods run on
+using tolerations on the pods.
+
+You apply tolerations to Elasticsearch pods through the Cluster Logging Custom Resource (CR)
+and apply taints to a node through the node specification. A taint on a node 
+instructs the node to repel all pods that do not tolerate the taint.
+
+By default, the Elasticsearch pods have the following toleration:
+
+[source, yaml]
+----
+tolerations:
+- effect: "NoExecute"
+  key: "node.kubernetes.io/disk-pressure"
+  operator: "Exists"
+----
+
+.Prerequisites
+
+* Cluster logging and Elasticsearch must be installed.
+
+.Procedure
+
+. Use the following command to add a taint to a node where you want the Elasticsearch pods scheduled:
```

Comment:
i'm not sure if this is the proper use of tainting nodes...
we would want to add a toleration if we needed to ensure that the component's pod can be deployed no matter what

Reply:
Changed to: _Use the following command to add taints and tolerations for cluster logging_
I added to the intro paragraph the use case in the Jira:
_You can use taints and tolerations to ensure that cluster logging pods 
run on specific nodes and that no other workload can run on those nodes._